docs/project-incubation-exit.md

```diff
@@ -0,0 +1,139 @@
+not and this decision does not need to be solely based on these exit
+criteria. The purpose of these exit criteria is to help the TSC in its
+decision process by informing its members of key aspects of the project.
+
+# Minimum requirements
+
+-   Legal
+
+    -   All code has been made available under the Apache License and is
+        free of incompatible dependencies
+    -   Project name has been checked for trademark issues
+
+-   Community support
+
+    -   The project must have an active and diverse set of contributing
+        members representing various constituencies
+    -   The project is not highly dependent on any single contributor
+        (there are at least 3 legally independent committers and there
+        is no single company or entity that is vital to the success of
+        the project)
+    -   Release plans are developed and executed in public by the
+        community.
+
+-   Sufficient test coverage
+
+    The project must include a comprehensive unit and integration test
+    suite and document its coverage. Additional performance and scale
+    test capability is desirable.
+
+-   Sufficient user documentation
+
+    The project must including enough documentation for anyone to test
+    or deploy any of the modules.
+
+-   Alignment
+
+    -   Requirements fulfillment
+
+        The project must document what requirements and use cases it addresses.
+
+    -   Architecture
+
+        The project must document how it fits within the [Hyperledger Architecture](https://wiki.hyperledger.org/display/AWG)
+
+    -   Compatibility with other Hyperledger projects
+
+        Where applicable, the project should be compatible with other
+        active projects.
+
+    -   Release numbering: the project should use the Hyperledger
+        standard [release taxonomy](./release-taxonomy.md), once that is agreed upon.
+
+    -   Project must make a [release](./release-taxonomy.md), even a "developer preview",
+        before graduation.
+
+-   Infrastructure
+
+    -   Github repo has been created
+    -   Mailing lists have been created and are archived
+    -   Other communication means used, such as chat channels, are set up
+    -   Project is set up with Continuous Integration
+    -   All information necessary for someone to join the community and
+        be able to start contributing is duly documented (location of
+        repo, list of maintainers, mailing lists addresses, slack
+        channels if used, etc) following Hyperledger standard practice
+        (CONTRIBUTING.md, MAINTAINERS.txt, etc)
+
+-   CII Badge
+
+    A team seeking to graduate from incubation shall have started the
+    CII Badge application and be nearly complete with incomplete badge
+    requirements referenced in their graduation proposal. 100% of the
+    applicable criteria for the CII Badge is a requirement for releasing
+    a 1.0 of the project. That does not mean the project must have 100%
```

Comment:
I think "1.0" should be replaced by "major release within Hyperledger" so that it works for projects that start with a higher version number and have had major releases before moving to Hyperledger.